test/tests/httptests.ts

```diff
@@ -37,13 +37,34 @@ describe('Http Tests', function () {
     //     }, 
     //     "origin": "173.95.152.44", 
     //     "url": "https://httpbin.org/get"
-    //  }  
+    //  }
     it('does basic http get request', async() => {
         let res: httpm.HttpClientResponse = await _http.get('http://httpbin.org/get');
         assert(res.message.statusCode == 200, "status code should be 200");
         let body: string = await res.readBody();      
         let obj:any = JSON.parse(body);
         assert(obj.url === "https://httpbin.org/get");
+        assert('User-Agent' in obj.headers === true, "User-Agent should be set");
+    });
+
+    it('does basic http get request with null agent', async() => {
+        let http: httpm.HttpClient = new httpm.HttpClient(null);
```

Comment:
Nit: can use const for all of these.

Reply:
no other const in the file. maybe ok for now?